demo-automation-java/src/test/java/com/example/pages/CarReviewPage.java

```diff
@@ -3,4 +3,5 @@
 public class CarReviewPage {
     
 }
-//code cai gi do
\ No newline at end of file
+jkhsdkfsdf
```

Comment:
code cho nay soi oi

Reply:
This line have been changed in last commit. Please review if you have any suggestions